microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+1. Click **Start instrumenting coverage and refresh the page** (![Refresh icon](../media/reload-icon.msft.png)) to display the coverage report, and then type `css` in the **URL filter** field to only display the CSS files:
 
-1. Click **Refresh** (![Refresh](../media/reload-icon.msft.png)).  The **Coverage** tool provides an overview of how much of the code in `bundle.js`, `jquery.js`, and `lodash.js` runs while the page loads.  In the figure after the following, about 76% and 30% of the jQuery and Lodash files aren't used, respectively.
+   ![The new Coverage report, now showing the CSS files](./images/coverage-report-css.png)
 
-   ![The Coverage report](../media/speed-glitch-tony-remix-updated-audits-performance-oppportunities-expanded-drawer-coverage-reloaded.msft.png)
+    The report shows that the `contact-form.css` and `gallery.css` files aren't used at all. They both have 100% of unused bytes.
 
-1. Click the `jquery.js` row.  DevTools opens the file in the **Sources** tool.  If a line of code ran, a blue bar appears next to it.  A red bar means the line of code was not run, and is definitely not needed on load of the webpage.
+1. Click the `contact-form.css` file in the report.  DevTools opens the file in the **Sources** tool.  If a line of code ran, a blue bar appears next to it.  A red bar means the line of code didn't run, and is definitely not needed on load of the webpage.
 
-   ![Viewing the jQuery file in the Sources tool](../media/speed-glitch-tony-remix-updated-sources-drawer-coverage-reloaded-jquery-js.msft.png)
+   ![The contact-form.css file in the Sources tool, with red bars next to the unused lines](./images/unused-css-source.png)
 
-1. Scroll through the jQuery code.  Some of the lines that run are actually just comments.  To strip comments and reduce the size of the file, run the code through a minifier app or script.
+   Only red bars are displayed in this source file, which means that the webpage does not need this file at all. 
 
-In short, when you're working with your own code, the **Coverage** tool helps you analyze your code, line-by-line, and only ship the code that's needed for page load.
+Now, remove the references to these files from the code:
 
-Are the `jquery.js` and `lodash.js` files even needed to load the page?  The **Request blocking** tool shows what happens when resources aren't available:
+1. In Visual Studio Code, open the `/travel-site/index.html` file.
 
-1. Select the **Network** tool.
+1. Near the top of the file, find the list of `<link>` tags that load the stylesheets on the page.
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the Command Menu again.
+1. Delete the two lines of code that load the `contact-form.css` and `gallery.css` files:
 
-1. Start typing `blocking`, and then select **Show Request Blocking**.
+    ![VS Code, with the index.html opened, showing where the two link tags are](./images/delete-unused-css.png)
```

Comment:
```suggestion
    ![Visual Studio Code, with the index.html opened, showing where the two link tags are](./images/delete-unused-css.png)
```
GLOBAL in PR: Find "VS Code".  (though the official docs sometimes write "VS Code")